### ISSUE TITLE
pflotran ignores FFLAGS in TPL build

### ISSUE THREAD

**Issue by ecoon** [amanzi/amanzi#745]
To set FFLAGS in PFLOTRAN, SuperBuild writes a script, 

```❯ cat pflotran-build-step.sh
#!/bin/bash

# CMake generates this file
# Replaces each  with value of ${VAR}

# Ensure a uniform build with the correct compiler
export CC=/usr/local/bin/mpicc
#export CFLAGS=''
export CXX=/usr/local/bin/mpicxx
#export CXXFLAGS=''
export F77=/usr/local/bin/mpifort
export FFLAGS=' -fPIC -fallow-argument-mismatch -O2 -g -DNDEBUG'
export PETSC_DIR='/Users/Shared/ornldev/code/ats/amanzi-tpls/install/master/local/openmpi-4.1.3/relwithdebinfo/petsc-3.16'

# pflotran in-source build under src
# NOTE: PFlotran doesn't seem parallel-build safe, since it has F90
# NOTE: modules that need to be built before other things, and dependencies
# NOTE: aren't expressed properly. The -j1 flag ensures that the build is
# NOTE: serial.
cd pflotran/pflotran-3.0.2-source/src/pflotran
make -j1 pflotran_rxn
rc=$?
echo "make pflotran_rxn returned $rc"

exit $rc
```

Unfortunately, PFLOTRAN's pflotran-3.0.2-source/src/pflotran/makefile doesn't look for FFLAGS, it simply sets that to empty:

``` line 203:
# These flags are supplemental to the PETSc flags
CFLAGS   =
FFLAGS   =
CPPFLAGS = ${MYFLAGS}
FPPFLAGS = ${MYFLAGS}
```

So maybe this is ok?  If PETSc gets the right flags, then maybe PFloTran will get them too?  I was hacking on getting "-fallow-argument-mismatch" into pflotran to fix a 1.3 build using gcc 10+ (which made this an error) and tried to just fix pflotran without getting the flag into PETSc, which doesn't need it.  But we should probably stop trying to set FFLAGS in PFLOTRAN this way... it doesn't work.

**Comment by jd-moulton** [comment]
This was fixed on master after the 1.3 release (#658).  Should just cherry pick the update to 1.3.

**Comment by ecoon** [comment]
sure, that fix makes sense.  Thanks!
